src/api/c/blas.cpp

```diff
@@ -239,6 +237,14 @@ af_err af_matmul(af_array *out, const af_array lhs, const af_array rhs,
 
         af_dtype lhs_type = lhsInfo.getType();
         switch (lhs_type) {
+#ifndef AF_CPU
+            case f16: {
+                    static const af_half alpha = { 0x03c00 }; // 1.0 : 0 01111 0000000000
```

Comment:
perhaps to be moved in some headers

Reply:
Internally we are using common::half which have constructors for floats.
```suggestion
                    static const half alpha(1.0f);
```

Reply:
done